[#46439] Bahn Plugin für OpenProject 12.5.x anpassen

ba1ash opened the pull request:
https://community.openproject.org/work_packages/46439

ba1ash committed:
[#46439] Bahn Plugin für OpenProject 12.5.x anpassen

https://community.openproject.org/work_packages/46439

Kharonus commented on frontend/src/app/shared/components/attachments/attachment-list/attachment-list-item.component.ts:
```diff
@@ -167,6 +167,13 @@ export class OpAttachmentListItemComponent extends UntilDestroyedMixin implement
+    }
+    return this.attachment._links.staticDownloadLocation.href;
+  }
+
```
I'd probably remove this extra getter and instead just use in the template:

```html
[href]="attachment._links.originOpen?.href || attachment._links.staticDownloadLocation.href"
```

ba1ash replied:
Cool!

Kharonus commented on frontend/src/app/shared/components/attachments/attachment-list/attachment-list-item.component.html:
```diff
@@ -28,10 +28,18 @@
+      <span class="spot-icon spot-icon_download-arrow"></span>
+    </a>
     <button
+      *ngIf="!!this.attachment._links.delete"
```
you do not need to `this.` in the template

Kharonus reviewed (changes requested):
I'd like to reduce a bit the code used in here.

ba1ash committed:
Refactor code a bit.

- Replace getter with more simple inline implementation.
- No need to use `this` in template. Remove it.

Kharonus reviewed (approved):
LGTM
